src/app/modules/main/wallet_section/saved_addresses/module.nim

```diff
@@ -42,6 +40,13 @@ method load*(self: Module) =
     ))
   )
 
+method load*(self: Module) =
+  singletonInstance.engine.setRootContextProperty("walletSectionSavedAddresses", newQVariant(self.view))
+
+  self.events.on("savedAddress/changed") do(e:Args):
+    self.loadSavedAddresses()
```

Comment:
Events should be in the controller

Reply:
Did it cause you asked gently but to be clear.

You asked me to pass eventemitter down, so that I could only call the same function up and as such create an extra binding between 2 objects that were not really (ok at all) necessary?

Reply:
That's the way the architecture was created 🤷 I think it's because only the controller has to know about the service and such, then the module can handle whatever the controller got from the event. 
I do understand that it's a bit redundant, especially in your case, but I'm just repeating what has been told to me 😬